values.yaml

```diff
@@ -45,14 +45,8 @@ global:
   imagePullSecrets: []
 
   # imageK8S is the name (and tag) of the consul-k8s Docker image that
-  # is used for functionality such as catalog sync. This can be overridden
+  # is used for functionality such as catalog sync and connect. This can be overridden
   # per component.
-  # Note: support for the catalog sync's liveness and readiness probes was added
-  # to consul-k8s 0.6.0. If using an older consul-k8s version, you may need to
-  # remove these checks to make the sync work.
-  # If using acls.manageSystemACLs then must be >= 0.10.1.
-  # If using connect inject then must be >= 0.10.1.
-  # If using Consul Enterprise namespaces, must be >= 0.12.
   imageK8S: "hashicorp/consul-k8s:0.20.0"
```

Comment:
todo: use 0.20.1

I removed the comments about correct k8s versions because they're definitely out of date and I don't think it's sustainable to keep updating them.